ui/Screen/Org.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="typescript">
+  import * as router from "ui/src/router";
+  import * as ipc from "ui/src/ipc";
+  import * as notification from "ui/src/notification";
+  import * as org from "ui/src/org";
+  import { unreachable } from "ui/src/unreachable";
+
+  import { Icon } from "ui/DesignSystem/Primitive";
+  import {
+    ActionBar,
+    AdditionalActionsDropdown,
+    FollowToggle,
+    Header,
+    SidebarLayout,
+    TabBar,
+  } from "ui/DesignSystem/Component";
+
+  import ProjectsTab from "ui/Screen/Org/Projects.svelte";
+  import MembersTab from "ui/Screen/Org/Members.svelte";
+  import OrgHeader from "ui/Screen/Org/OrgHeader.svelte";
+  import ProjectsMenu from "ui/Screen/Org/ProjectsMenu.svelte";
+  import MembersMenu from "ui/Screen/Org/MembersMenu.svelte";
+
+  export let activeTab: router.LoadedOrgTab;
+  export let gnosisSafeAddress: string;
+  export let address: string;
+  export let members: org.Member[];
+  export let threshold: number;
+
+  const tabs = (address: string, active: router.LoadedOrgTab) => {
+    return [
+      {
+        title: "Anchored Projects",
+        icon: Icon.ChevronLeftRight,
+        active: active.type === "projects",
+        onClick: () => {
+          router.push({ type: "org", activeTab: "projects", address });
+        },
+      },
+      {
+        title: "Members",
+        icon: Icon.User,
+        active: active.type === "members",
+        counter: members.length,
+        onClick: () => {
+          router.push({ type: "org", activeTab: "members", address });
+        },
+      },
+    ];
+  };
+
+  const copy = (content: string): void => {
+    if (content) ipc.copyToClipboard(content.trim());
```

Comment:
`content` isn’t empty so I don’t think we need this check. Maybe we can also inline this function?

Reply:
b704f566f7a44970352ae71b5df8e2bfcc88fbb8